cmd/context/update-kubeconfig.go

```diff
@@ -35,40 +46,65 @@ func UpdateKubeconfigCMD() *cobra.Command {
 		RunE: func(cmd *cobra.Command, args []string) error {
 			ctx := context.Background()
 
+			// Run context command to get the Cfg into Okteto Context
 			if err := NewContextCommand().Run(ctx, &ContextOptions{}); err != nil {
 				return err
 			}
 
-			if err := ExecuteUpdateKubeconfig(); err != nil {
+			okCtx := okteto.Context()
+			kubeconfigPath := config.GetKubeconfigPath()
+			kubetokenEnabled := false
+
+			okClient, err := okteto.NewOktetoClient()
+			if err != nil {
 				return err
 			}
-
-			return nil
+			if err := okClient.Kubetoken().CheckService(okCtx.Name, okCtx.Namespace); err == nil {
```

Comment:
Should we also check here the feature flag we are adding in other PR?

Reply:
i believe we will on the PR for feature flag 

cc: @andreafalzetti

Reply:
The feature flag is covered by https://github.com/okteto/app/issues/6886 so as soon as this PR is merged, I can rebase and finish the [feature flag PR](https://github.com/okteto/okteto/pull/3893)